AsaApi/Core/Private/Tools/Requests.cpp

```diff
@@ -2,6 +2,7 @@
 #define WIN32_LEAN_AND_MEAN
 #pragma warning(push)
 #pragma warning(disable: 4191)
+#pragma warning(disable : 4996)
```

Comment:
What deprecated functionality are we using?

Reply:
I flagged the old POST's as deprecated which do not supply the return headers. I figured eventually we may want to remove them.